build.yaml

```diff
@@ -513,9 +599,9 @@ common_cli:
   # By doing this we speed up the bootstrapping of our client/acceptance tests.
   #
   build_dockerfile_centos7: &build_dockerfile_centos7 |
-    # Download the latest pip requirements file from master branch of flocker
+    # Download the latest pip requirements file from flocker
     wget -c \
-    https://raw.githubusercontent.com/ClusterHQ/flocker/master/requirements.txt
+    https://raw.githubusercontent.com/ClusterHQ/flocker/\${TRIGGERED_BRANCH}/requirements.txt
```

Comment:
No change required, but just curious if this fixes a bug or something? It looks like a really good idea.